ci-scripts/release-tests.sh

```diff
@@ -132,12 +132,12 @@ function storage_limit_test() {
 }
 
 # !!! Configure here !!!
-VERSION_OLD="1.4"
-VERSION_NEW="1.5"
-RHDH_HELM_REPO_OLD="https://charts.openshift.io/"
-RHDH_HELM_REPO_NEW="https://raw.githubusercontent.com/rhdh-bot/openshift-helm-charts/refs/heads/redhat-developer-hub-1.5-147-CI/installation"
-SOURCE_BRANCH_OLD=rhdh-v1.4.x
+VERSION_OLD="1.5"
+VERSION_NEW="1.6"
+RHDH_HELM_REPO_OLD="https://raw.githubusercontent.com/rhdh-bot/openshift-helm-charts/refs/heads/redhat-developer-hub-1.5-148-CI/installation"
```

Comment:
@jhutar the version for 1.5 should be `1.5-178`

Reply:
Thank you, fixed.